src/test/scripts/get_product_lts_version.py

```diff
@@ -0,0 +1,63 @@
+import sys
+import urllib.request
+import json
+
+known_lts_version = {
+	'jira-software':'8.13.7',
```

Comment:
Assuming this map should be using the latest LTS product versions? If so the current LTS for Jira is `8.13.8`.